docs/application/tizen-studio/rt-ide/overview.md

```diff
@@ -1,24 +1,16 @@
-# Tizen RT
+# TizenRT
 
-Tizen has commercialized in smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of Tizen RT is to extend the Tizen platform device coverage to these kind of low-end devices.
+Tizen has commercialized in smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of TizenRT is to extend the Tizen platform device coverage to these kind of low-end devices.
 
-Tizen RT is an RTOS-based lightweight platform. The following are the hardware and environmental requirements:
+TizenRT is an RTOS-based lightweight platform. The following are the hardware and environmental requirements:
 
 - Hardware targets: 
 
   Cortex-M/R processors with MPU, less than 2 MB RAM, and less than 16 MB Flash.
 
 - Development environment:
-	- Tizen RT Studio for SmartThings:
-	Tizen RT Studio version 2.0 (or later) is used as an IDE for developing IoT applications with the Tizen IoT Platform and ARTIK 053(s).
+	- TizenRT Studio for SmartThings:
+	TizenRT Studio version 2.0 (or later) is used as an IDE for developing IoT applications with st_things framework and ARTIK 053(s).
 
-	- IoT Setup Wizard:
-	The Tizen RT Setup Wizard is a computer application for easily setting up Tizen IoT on hardware targets. In the Tizen OS version 4.0, flashing a platform image into ARTIK 053(s) is only supported with the IoT Setup Wizard in a Linux computer environment.
-
-To start developing your Tizen IoT applications:
-1. [Install the Tizen Studio for RT](getting-started/install.md)
-2. [Manage the project](getting-started/create.md)
-3. [Managing SmartThings&trade; Certificates](smartthings-ext/certificate-smartthings.md)
-4. [Flash the project](getting-started/flash.md)
-5. [Use the serial terminal](getting-started/terminal.md)
-6. [Debug the project](getting-started/debug.md)
+For more information on TizenRT, please refer to GitHub
```

Comment:
How about changing line 15 and 16 into `For more information on TizenRT, please refer to [TizenRT GitHub](https://github.com/Samsung/TizenRT)`?

Reply:
Ok. i will change it.

Reply:
In the updated version, we have `#` in front of `TizenRT GitHub`. If it is not necessary, how about deleting it?

![image](https://user-images.githubusercontent.com/9958374/39339083-badba340-4a01-11e8-87a9-9598e5516ce0.png)